matminer/featurizers/structure.py

```diff
@@ -3612,3 +3613,189 @@ def citations(self):
     def implementors(self):
         return ["Alex Dunn", "Lauri Himanen"]
 
+    
+class GlobalInstabilityIndex(BaseFeaturizer):
+    '''
+    Will compute the global instability index of a structure.
+    
+    The default is to use IUCr 2016 bond valence parameters for computing 
+    bond valence sums. If the structure has disordered site occupancies 
+    or non-integer valences on sites, pymatgen's bond valence sum method 
+    can be used instead. 
+    
+    Note that pymatgen's bond valence sum method is prone to error unless 
+    the correct scale factor is supplied. A scale factor based on testing 
+    with perovskites is used here.
+    TODO: Use scipy to optimize scale factor for minimizing GII
+    
+    Based on the following publication:
+    
+    "Structural characterization of R2BaCuO5 (R = Y, Lu, Yb, Tm, Er, Ho, 
+        Dy, Gd, Eu and Sm) oxides by X-ray and neutron diffraction", 
+        A.Salinas-Sanchez, J.L.Garcia-Muñoz, J.Rodriguez-Carvajal,
+        R.Saez-Puche, and J.L.Martinez, Journal of Solid State Chemistry,
+        100, 201-211 (1992),
+        https://doi.org/10.1016/0022-4596(92)90094-C
+        
+    Args:
+        r_cut: Float, how far to search for neighbors when computing bond valences
+        disordered_pymatgen: Boolean, whether to fall back on pymatgen's bond 
+            valence sum method for disordered structures
+    '''
+    
+    def __init__(self, r_cut=4.0, disordered_pymatgen=False):
+        
+        basedir = os.path.dirname(os.path.realpath(__file__))
+        bv_path = os.path.join(basedir, "../utils/data_files/Bond_valences2016.csv")
```

Comment:
I'm concerned that using both a path with `/` and `os.path.join` will cause problems for Windows users. Could you break the path up into individual folders/paths (i.e., `(basedir, '..', 'utils', ...)`)?